src/context/viewer-context.tsx

```diff
@@ -56,6 +56,21 @@ function useAuthedViewer() {
     const accessToken = get(queryHash, 'access_token')
     const noAccessTokenFound = isEmpty(accessToken)
     const viewerIsPresent = !isEmpty(viewerId)
+    const authToken = getAccessTokenFromCookie()
+
+    console.log({loggingOut})
+
+    if (loggingOut) {
```

Comment:
since `loggingOut` is a state, we can capture it in the effect and make sure we aren't running anything else if that's what we are doing right now